src/keria/app/cli/commands/start.py

```diff
@@ -111,8 +112,15 @@ def launch(args):
                             bootPassword=args.bootPassword,
                             bootUsername=args.bootUsername)
 
-    directing.runController(doers=agency, expire=0.0)
+    tock = 0.03125
+    doist = doing.Doist(limit=0.0, tock=tock, real=True)
```

Comment:
The `directing.runController` did not return the `doist`, so there was no way to access it to be able to call `exit`. However, it was only two lines of code, so I thought we could just inline it here.

Reply:
This makes sense. This is good enough for now. We could consider making a PR to change KERIpy to have `runController` either return the `Doist` or have a parameter to pass a shutdown signal.